src/System.IO.FileSystem/tests/Base/FileGetSetAttributes.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.IO.Tests
+{
+    // Tests that are valid for File and FileInfo
+    public abstract class FileGetSetAttributes : BaseGetSetAttributes
+    {
+        [Theory]
+        [InlineData(FileAttributes.ReadOnly)]
+        [InlineData(FileAttributes.Normal)]
+        [PlatformSpecific(TestPlatforms.AnyUnix)]  // Unix valid file attributes
+        public void UnixAttributeSetting(FileAttributes attr)
+        {
+            var path = CreateItem();
+            Set(path, attr);
+            Assert.Equal(attr, Get(path));
+            Set(path, 0);
+        }
+
+        [Theory]
+        [InlineData(FileAttributes.ReadOnly)]
+        [InlineData(FileAttributes.Hidden)]
+        [InlineData(FileAttributes.System)]
+        [InlineData(FileAttributes.Archive)]
+        [InlineData(FileAttributes.Normal)]
+        [InlineData(FileAttributes.Temporary)]
+        [InlineData(FileAttributes.ReadOnly | FileAttributes.Hidden)]
+        [PlatformSpecific(TestPlatforms.Windows)]  // Valid Windows file attribute
+        public void WindowsAttributeSetting(FileAttributes attr)
+        {
+            var path = CreateItem();
+            Set(path, attr);
+            Assert.Equal(attr, Get(path));
```

Comment:
"Get" is a very generic name; it's difficult when reading this code to know what we're getting.  Please expand the name to be more descriptive.